orion/sites/admin/src/announce/announce.styl

```diff
@@ -5,9 +5,48 @@
   height: 400px
   padding: 0 spacing2x
   box-sizing: border-box
-  color: colorBlackDark
+  color: colorGrayDark
+  font-family: fontFamilyRoboto /*not working*/
 
   & h1
     margin: spacing4x 0
     line-height: lineHeight3
     font-size: fontSize3
+  & ul
+    padding: 0
+
+  & .announce-lists
+    display: inline-block
+    list-style: none
+    padding-left: 0
+    height: lineHeight6
+    line-height: lineHeight6
+    border-bottom: 1px solid colorGrayLight
+
+    & li
+      display: inline-block
+    & div
+      display: inline-block
+    & p
+      display: inline-block
+
+#view-announce
+      & .li-small
+        width: spacing8x
+        color: colorTurquoise
+
+      & .li-med
+          width: spacing16x
+
+      & .li-large
+          width: spacing64x
+
+  .announcement-button
+    display: inline-block
+    color: colorWhite
+    background-color: colorTurquoise
+    width: spacing32x
+    height: lineHeight6
+    line-height: lineHeight6
+    text-align: center
+    border-radius: spacing1x
```

Comment:
Look at the comment above about CSS property ordering
This can be re-ordered as:
```
display: inline-block
width: 
height
line-height
border-radius
text-align
color
background-color
```

The point of doing this is to easily see what properties are being styled on this class/element. When it's a random order, you'll probably miss a few lines and end up adding 2 of the same properties.
Another thing is the properties try to order themselves from most significant properties (display / position, etc.) to least significant properties (like colors). When the properties in a categorized order, they become much easier to debug!